Extensions/BBText/bbtextruntimeobject.js

```diff
@@ -1,34 +1,53 @@
+//TODO @blurrymind correct the descriptions.
+/**
+ * @typedef {Object} BBTextObjectDataType Base parameters for {@link gdjs.BBTextRuntimeObject}
+ * @property {Object} content The base parameters of the BBText
+ * @property {number} content.opacity The opacity of the BBText
+ * @property {boolean} content.visible Is the text visible?
+ * @property {string} content.text Text of the object
+ * @property {string} content.color The color of the object
+ * @property {string} content.fontFamily The Font of the object
+ * @property {number} content.fontSize The size of the Text of the object
+ * @property {boolean} content.wordWrap Toggle Word Warp
+ * @property {string} content.align Align? //TODO correct that
+ * 
+ * @typedef {ObjectData & BBTextObjectDataType} BBTextObjectData
+ */
+
 /**
  * Displays a rich text using BBCode markup (allowing to set parts of the text as bold, italic, use different colors and shadows).
  * @memberof gdjs
  * @class BBTextRuntimeObject
  * @extends RuntimeObject
+ * @param {gdjs.runtimeScene} runtimeScene The {@link gdjs.RuntimeScene} the object belongs to
+ * @param {BBTextObjectData} objectData The optional object parameters
```

Comment:
Let's remove optional from the description of the parameter as it's not optional.
I may have put "optional" for these kind of parameters for other objects => don't hesitate to delete "optional" if you see it in other objects :)

Reply:
corriged